src/test/java/tests/UserTest.java

```diff
@@ -0,0 +1,48 @@
+package tests;
+
+import configurations.BaseTest;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+
+import org.hamcrest.Matchers;
+
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static enums.ResponseCodes.OK;
+import static java.util.Collections.emptyMap;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.testng.AssertJUnit.assertFalse;
+
+@Slf4j
+public class UserTest extends BaseTest {
+
+  @Test
+  public void getStatus() {
+    Response response =
+            restClient.sendRequestWithParams(Method.GET, emptyMap());
+
+    assertThat(response.statusCode(), Matchers.equalTo(OK.getValue()));
+  }
+
+  @Test
+  public void getHeader() {
+    Response response =
+            restClient.sendRequestWithParams(Method.GET, emptyMap());
+
+    assertFalse(response.getHeader(headerName).isEmpty());
+    assertThat(response.getHeader(headerName),
+            Matchers.equalTo(headerValue));
+  }
+
+  @Test
+  public void getCountOfUser() {
+    Response response =
+            restClient.sendRequestWithParams(Method.GET, emptyMap());
+    List<String> userList = response.jsonPath().getList("");
+
+    assertThat(userList.size(), Matchers.equalTo(usersCount));
```

Comment:
Test data shouldn't be located in properties. It should be inside the test case or in a separate test data file (preferable).

Only some general values can be put into properties. Fox example, the default wait time or test data templates path, etc.